das-query-engine/tests/integration/handle/base_test_action.py

```diff
@@ -3,8 +3,9 @@
 
 import pytest
 from handler import UnreachableConnection, handle
-
-
+@pytest.mark.skip(
+    reason="These tests need to be refactored"
```

Comment:
@marcocapozzoli  @levisingularity 

May I propose to follow this procedure whenever we disable some test:

1 - Create a card to fix whatever is the issue which motivated the disabling of the test and set its priority accordingly
2 - Disable the test with `reason="Disabled because of repo#XXX"` where repo is the repository where the issue has been created and XXX is the card number.

Reply:
@andre-senna I think it's a good idea.

Reply:
I agree